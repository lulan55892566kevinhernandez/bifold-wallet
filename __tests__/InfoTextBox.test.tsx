import React from 'react'
import renderer from 'react-test-renderer'

import InfoTextBox from '../App/components/texts/InfoTextBox'

jest.mock('react-native-vector-icons/MaterialIcons', () => 'Icon')

describe('InfoTextBox Component', () => {
  it('Renders correctly', () => {
    const tree = renderer
      // @ts-ignore
      .create(<InfoTextBox text="Hello World" />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
