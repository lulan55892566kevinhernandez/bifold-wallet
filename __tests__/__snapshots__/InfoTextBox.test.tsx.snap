// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoTextBox Component Renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#D9EAF7",
        "borderColor": "#B9CEDE",
        "borderRadius": 5,
        "borderWidth": 1,
        "flexDirection": "row",
        "padding": 10,
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "marginRight": 10,
        },
      ]
    }
  >
    <Icon
      color="#313132"
      name="info"
      size={30}
    />
  </View>
  <Text
    style={
      Array [
        Object {
          "color": "#313132",
          "flexShrink": 1,
          "fontSize": 18,
          "fontWeight": "normal",
        },
        Object {
          "fontWeight": "bold",
        },
      ]
    }
  />
</View>
`;
